### PR TITLE
Front-end tweaks

### PR DESCRIPTION
**Pull request by samdutton**
• Tweaked text.
• Made visual styles more like web.dev.
• Layout is mobile-friendly (for reading only!)
• Moved CMD into index.html and used simpler syntax.
• Added copy-code button script, but you may want to put this in a separate file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,74 +1,106 @@
...
+      
+      <section>
+        <p>1. Open Chrome Canary 87 (or later) from a terminal. On MacOS use the following command:</p>
+          <div id="code">
+            <svg id="copy" height="24" width="24">
+              <title>Copy code</title>
+              <path d="M0 0h24v24H0z" fill="none"/>
+              <path d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm3 4H8c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h11c1.1 0 2-.9 2-2V7c0-1.1-.9-2-2-2zm0 16H8V7h11v14z"/>
+            </svg>
+            <pre id="command">
```
Comment (JackJey) [inline index.html]: commands should be generated dynamically so this should no be written statically here.

File: public/js/issuer.js [hunk 2]
```diff
@@ -14,18 +14,6 @@ document.on("DOMContentLoaded", async e => {
     "/.well-known/trust-token/key-commitment"
   )).json();
 
-  // display chrome flag with key-commitment
-  const CMD = `/Applications/Google\\ Chrome\\ Canary.app/Contents/MacOS/Google\\ Chrome\\ Canary \\
-  --additional-trust-token-key-commitments='${JSON.stringify(COMMITMENT)}' \\
```
Comment (JackJey) [inline public/js/issuer.js]: if key pair of server updated, key commitment will change too.
so it should be generated dynamically from here. please don't remove.

**Review by JackJey — COMMENTED** [review]
Thanks Sam!
one thing from me is around CMD.

but it's ok for me to fixup them by myself,
so feel free to ping me merge as-is and update other commit for cover them from me.

File: index.html (hunk 1 above)
Comment (samdutton) [reply index.html]: Ah — I see.

File: public/js/issuer.js (hunk 2 above)
Comment (samdutton) [reply public/js/issuer.js]: Got it! I'll change this.

File: index.html (hunk 1 above)
Comment (samdutton) [reply index.html]: Reverted issuer.js to your version.

Comment (samdutton) [reply index.html]: BTW: Can we use the simpler syntax for running Chrome with flags on MacOS?

```open -a "Google Chrome Canary" --args \```

**Comment by JackJey** [comment]
LGTM !
I'll merge it and deploy soon.

**Pull request merged**